.vr_settings.yaml

```diff
@@ -1,18 +1,14 @@
 --- !ruby/object:VR_ENV
-width: 800
-height: 600
+width: 1113
+height: 666
 panel_pos: 360
 run_command_line: ruby tictactoe
 open_files:
-- "/Users/rcrjr/Flatiron/Prework/tictactoe_vr/README.md"
-- "/Users/rcrjr/Flatiron/Prework/tictactoe_vr/bin/player.rb"
-- "/Users/rcrjr/Flatiron/Prework/tictactoe_vr/bin/Board.rb"
-- "/Users/rcrjr/Flatiron/Prework/tictactoe_vr/bin/tictactoe.rb"
-- "/Users/rcrjr/Flatiron/Prework/tictactoe_vr/tictactoe"
+- "/home/eric/visualruby/tictactoe/tictactoe.gemspec"
```

Comment:
I think this path needs to be made more generic, or this file excluded from the project (added to `.gitignore`).